download.dd

```diff
@@ -149,7 +149,7 @@ $(DOWNLOAD_OTHER $(GENTOO), $(LINK2 https://wiki.gentoo.org/wiki/Dlang, Gentoo),
 
 $(DOWNLOAD_OTHER $(HOMEBREW), $(LINK2 https://formulae.brew.sh/formula/dmd, Homebrew), $(CONSOLE brew install dmd))
 
-$(DOWNLOAD_OTHER $(NIX), Nix/NixOS, $(CONSOLE nix-env -i dmd))
+$(DOWNLOAD_OTHER $(NIX), $(LINK2 https://search.nixos.org/packages?query=dmd Nix/NixOS), $(CONSOLE nix-env -iA nixpkgs.dmd))
```

Comment:
I'm not sure, but I think this syntax error was [preventing][0] `Nix/NixOS` to appear as a link:

[0]: http://dtest.dlang.io/artifact/website-def0043f44ddc3f6590931dcdc918839fd32fff7-7085a4d3fea84fbf28d1af71de9497b4/web/download.html

```suggestion
$(DOWNLOAD_OTHER $(NIX), $(LINK2 https://search.nixos.org/packages?query=dmd, Nix/NixOS), $(CONSOLE nix-env -iA nixpkgs.dmd))
```